cloudify_cli/commands/audit_log.py

```diff
@@ -0,0 +1,51 @@
+import click
+
+from ..cli import cfy, helptexts
+from ..table import print_data
+
+AUDITLOG_COLUMNS = ['ref_table', 'ref_id', 'operation', 'creator_name',
+                    'execution_id', 'created_at']
+
+
+@cfy.group(name='auditlogs')
```

Comment:
this `name` is the user-visible side, and in this, I think we should go for `auditlog` singular. In principle, it's a log, with log lines, not multiple logs, right?

(in practice, we'll all just use `cfy au li` anyway)